### DISCUSSION TITLE
[EN] Add the VAT rate for the product

### DISCUSSION THREAD

**Discussion by Valdnet** [baimard/gestion#202] [Ideas]
The products have a different VAT rate (this is the case in Poland).

I suggest that you can set different VAT rates in the application settings.
Then, when adding a product, you should add the VAT rate selection for the product.

![obraz](https://user-images.githubusercontent.com/47037905/143614210-e9aa294a-6fca-43da-bf5b-665272e984b0.png)

**Comment by baimard** [comment]
it's done
